src/compass/utils/h5_helpers.py

```diff
@@ -446,3 +427,80 @@ def poly1d_to_h5(group, poly1d_name, poly1d):
                           {'description': 'EPSG code identifying the coordinate system used for processing'})
     add_dataset_and_attrs(processing_group, 'no_data_value', 'NaN',
                           {'description': 'Value used when no data present'})
+
+
+def burst_corrections_to_hdf5(dst_h5, burst, cfg):
+    '''
+    Write azimuth, slant range, and EAP (if needed) correction LUT2ds to HDF5
+
+    Parameter:
+    ---------
+    dst_h5: h5py File
+        HDF5 file meta data will be written to
+    burst: Sentinel1BurstSlc
+        Burst containing corrections
+    cfg: types.SimpleNamespace
+        SimpleNamespace containing run configuration
+    '''
+    # geocoding correction LUTs
+    correction_group = dst_h5.require_group('corrections')
+
+    # Get range and azimuth LUTs
+    rg_lut, az_lut = compute_geocoding_correction_luts(burst,
+                                                       rg_step=cfg.lut_params.range_spacing,
+                                                       az_step=cfg.lut_params.azimuth_spacing)
+
+    desc = ' correction as a function of slant range and azimuth time'
+    add_dataset_and_attrs(correction_group, 'azimuth_LUT', az_lut.data,
+                          {'description': f'azimuth {desc}',
+                           'units': 'seconds'})
+    add_dataset_and_attrs(correction_group, 'slant_range_LUT', rg_lut.data,
+                          {'description': f'slant_range {desc}',
+                           'units': 'seconds'})
+
+    # az/rg_lut share the same grid
+    slant_range = np.linspace(az_lut.x_start,
+                              az_lut.x_start + az_lut.width * az_lut.x_spacing,
+                              az_lut.width, dtype=np.float64)
+    add_dataset_and_attrs(correction_group, 'slant_range', slant_range,
+                          {'description': 'slant range of LUT data',
+                           'units': 'meters'})
+
+    azimuth = np.linspace(az_lut.y_start,
+                          az_lut.y_start + az_lut.width * az_lut.x_spacing,
+                          az_lut.width, dtype=np.float64)
+    add_dataset_and_attrs(correction_group, 'azimuth_time', azimuth,
+                          {'description': 'azimuth time of LUT data',
+                           'units': 'seconds'})
+
+    # EAP metadata depending on IPF version
+    check_eap = is_eap_correction_necessary(burst.ipf_version)
+    if check_eap.phase_correction:
+        eap_group = correction_group.require_group('elevation_antenna_pattern_correction')
+        eap = burst.burst_eap
+        add_dataset_and_attrs(eap_group, 'sampling_frequency',
+                              eap.freq_sampling,
+                              {'description': 'range sampling frequency',
+                               'units': 'Hz'})
+        add_dataset_and_attrs(eap_group, 'eta_start',
+                              eap.eta_start.strftime(TIME_STR_FMT),
+                              {'description': '?',
```

Comment:
`eta_start` is sensing start time. `eta` came from the greek letter in the reference document [here](https://sentinels.copernicus.eu/documents/247904/1653440/Sentinel-1-IPF_EAP_Phase_correction.pdf/d8767e29-ae1d-4e1b-92c2-22c10648f81f?t=1437645629000) (section 5.3)